src/node_shell_read.js

```diff
@@ -4,17 +4,21 @@
  * SPDX-License-Identifier: MIT
  */
 
+requireNodeFS = function() {
+  if (!fs) fs = require('fs');
+  if (!nodePath) nodePath = require('path');
```

Comment:
```suggestion
  // We always initialize both of these together, so we can use
  // either one as the indicator for them not being initialized.
  if (!fs) {
    fs = require('fs');
    nodePath = require('path');
  }
```
This is I think smaller?

Reply:
I still see a 19 byte regression in this test (although we have 150 byte tollerance here). 

Its due to the function itself being part of the output:

```
E = function() { fs || (fs = require("fs"), D = require("path")) },
```

Since this is tiny and will not be present for folks that set `ENVIRONMENT=web` anyway I think its fine.

Reply:
None of actual code size tests regress even one byte.

Reply:
Ah right, good point that it just affects node-enabled builds.